lib/omniauth/strategies/artsy.rb

```diff
@@ -31,6 +31,16 @@ def self.configure
       def raw_info
         @raw_info ||= access_token.get('/api/current_user', headers: { 'X-ACCESS-TOKEN' => access_token.token }).parsed
       end
+
+      alias old_request_phase request_phase
```

Comment:
TIL you can do it with an alias. Is it effectively the same as calling `super`?

Reply:
I'm not 100% sure and was following [examples](https://github.com/cmply/omniauth-cmply/blob/054b7de5ea5ebb79475078ec944795a07741afa6/lib/omniauth/strategies/cmply.rb) for this.

I think they can be a little different. E.g., `super` might refer to the base class's implementation (`OmniAuth::Strategies::OAuth2` in this case), while an alias would refer to whatever `self.request_phase` resolves to at the time `alias` is called. Since the `Artsy` strategy can itself be extended, I thought it was safer to stick with this.